wp-content/plugins/jetpack/_inc/build/contact-form/js/editor-view.min.js

```diff
@@ -0,0 +1 @@
iting_style,r=t('<link rel="stylesheet" href="'+i+'" />'),c=t('<link rel="stylesheet" href="'+n.dashicons_css_url+'" />');r.on("load",(function(){a.contents().find("body").css("visibility","visible"),a.trigger("checkheight")})),a.contents().find("head").append(r).append(c),a.contents().find("body").html(e.mce.grunion_wp_view_renderer.editor_inline({to:s.attrs.named.to,subject:s.attrs.named.subject,fields:f})).css("visibility","hidden"),a.contents().find("input:first").focus(),setTimeout((function(){a.trigger("checkheight")}),250),setTimeout((function(){a.trigger("checkheight")}),500);var l=a.contents().find(".grunion-fields"),d=a.contents().find(".grunion-controls");l.sortable(),l.on("change select","select[name=type]",(function(){t(this).closest(".grunion-field-edit")[0].className="card is-compact grunion-field-edit grunion-field-"+t(this).val(),a.trigger("checkheight")})),l.on("click",".delete-option",(function(e){e.preventDefault(),t(this).closest("li").remove(),a.trigger("checkheight")})),l.on("click",".add-option",(function(n){var i=t(e.mce.grunion_wp_view_renderer.editor_option());n.preventDefault(),t(this).closest("li").before(i),a.trigger("checkheight"),i.find("input:first").focus()})),l.on("click",".delete-field",(function(e){e.preventDefault(),t(this).closest(".card").remove(),a.trigger("checkheight")})),d.find("input[name=submit]").on("click",(function(){var n=s;n.type="closed",n.attrs={},n.content="",l.children().each((function(){var i={tag:"contact-field",type:"single",attrs:{label:t(this).find("input[name=label]").val(),type:t(this).find("select[name=type]").val()}},o=[];t(this).find("input[name=required]:checked").length&&(i.attrs.required="1"),t(this).find("input[name=option]").each((function(){t(this).val()&&o.push(t(this).val())})),o.length&&(i.attrs.options=o.join(",")),n.content+=e.shortcode.string(i)})),a.contents().find("input[name=to]").val()&&(n.attrs.to=a.contents().find("input[name=to]").val()),a.contents().find("input[name=subject]").val()&&(n.attrs.subject=a.contents().find("input[name=subject]").val()),o(e.shortcode.string(n))})),d.find("input[name=cancel]").on("click",(function(){o(e.shortcode.string(s))})),d.find("input[name=add-field]").on("click",(function(){var n=t(e.mce.grunion_wp_view_renderer.edit_template({}));l.append(n),l.sortable("refresh"),a.trigger("checkheight"),n.find("input:first").focus()}))})),l.html(a)}},e.mce.views.register("contact-form",e.mce.grunion_wp_view_renderer),QTags.addButton("grunion_shortcode",n.labels.quicktags_label,(function(){QTags.insertContent("[contact-form]"+n.default_form+"[/contact-form]")}));var i=t("#wp-content-wrap");t("#insert-jetpack-contact-form").on("click",(function(t){t.preventDefault(),i.hasClass("tmce-active")?tinyMCE.execCommand("grunion_add_form"):i.hasClass("html-active")?QTags.insertContent("[contact-form]"+n.default_form+"[/contact-form]"):window.console.error("Neither TinyMCE nor QuickTags is active. Unable to insert form.")}))}}(jQuery,wp,grunionEditorView);
```

Comment:
:warning: **Warning( severity 5 )**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).